dedupe/tfidf.py

```diff
@@ -13,17 +13,24 @@ def __init__(self, field, stop_words=[]) :
         self._index = CanopyIndex(stop_words)
  
         self._i_to_id = {}
+        
+        self._id_to_i = {}
```

Comment:
hmm.. if we are going to keep this mapping around, then it would be much simper to just get rid of _hash32 and do something like

`self._id_to_i = ids = collections.defaultdict(itertools.count(-2**31).next)`